include/seqan3/core/char_operations/predicate.hpp

```diff
@@ -6,52 +6,52 @@
 // -----------------------------------------------------------------------------------------------------
 
 /*!\file
- * \brief Provides parse conditions for tokenization.
+ * \brief Provides Character predicates for tokenisation.
```

Comment:
```suggestion
 * \brief Provides character predicates for tokenisation.
```

This was only partially resolved